backend/Jenkinsfile

```
@@ -0,0 +1,46 @@
+pipeline
+{
+    agent any
+
+    tools {
+        gradle 'gradle'
+    }
+
+    stages
+    {
+        stage ('Checkout')
+        {
+            steps {
+                git branch: 'test/jenkins', url: 'https://github.com/woowacourse-teams/2022-pickpick.git'
+            }
+        }
+        stage ('Build')
+        {
+            steps {
+                sh '''cd ./backend
+                chmod +x ./gradlew
+                ./gradlew bootJar
+                cd ..
+'''
```

```suggestion
                cd ..'''
```
또는
```suggestion
                cd ..
                '''
```
`'''`이 친구도 정렬 맞춰주면 좋을 것 같아요 😄